libraries/adsk/shaders/brick/ND_brick_02_shader.mtlx

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<materialx version="1.37">
+  <nodedef name="ND_brick_02_surfaceshader" node="brick_02" nodegroup="shader" version="1.0" isdefaultversion="true" namespace="adsk">
+     <parameter name="ao_image" type="filename" value="adsk/textures/bricks_02/2k/bricks_02_ao_2k_raw.exr" />
```

Comment:
Expose at this level so it shows up as uniforms which can be queried and modified if needed.